bundles/org.openhab.binding.boschindego/src/main/java/org/openhab/binding/boschindego/internal/console/BoschIndegoCommandExtension.java

```diff
@@ -84,8 +112,46 @@ public List<String> getUsages() {
     @Override
     public boolean complete(String[] args, int cursorArgumentIndex, int cursorPosition, List<String> candidates) {
         if (cursorArgumentIndex <= 0) {
+            return new StringsCompleter(
+                    Stream.concat(Stream.of(AUTHORIZE), getBridgeIds().stream().map(ThingUID::getAsString)).toList(),
+                    false).complete(args, cursorArgumentIndex, cursorPosition, candidates);
+        } else if (cursorArgumentIndex == 1 && !AUTHORIZE.equals(args[0])) {
             return SUBCMD_COMPLETER.complete(args, cursorArgumentIndex, cursorPosition, candidates);
         }
         return false;
     }
+
+    private List<ThingUID> getBridgeIds() {
+        List<ThingUID> bridgeIds = new ArrayList<>();
+        for (Thing thing : thingRegistry.getAll()) {
+            if (thing.getHandler() instanceof BoschAccountHandler) {
+                bridgeIds.add(thing.getUID());
```

Comment:
I'm not sure if you intended to use bridge **ID** (e.g. `singlekey`) as the naming suggests, or **UID** (e.g. `boschindego:account:singlekey`)? This needs to be consistent. I guess ID would be sufficient with less typing:
```suggestion
                bridgeIds.add(thing.getUID().getId());
```

Reply:
Will change to use bridge Id instead of UID. I didn't know the exact structure of the UID.